composer.json

```diff
@@ -18,15 +18,12 @@
     "require": {
         "composer/installers": "*",
         "silverstripe/framework": "~3.1",
-        "madmatt/simplesamlphp": "dev-ss-master"
+        "onelogin/php-saml": "v2.10.2"
```

Comment:
I'd urge against locking to a single version - this makes taking in security updates from the upstream harder and can be frustrating if you rely on a bug fix in a subsequent version.

A project maintainer should be given the decision to lock onto a version, rather than the module.